### DISCUSSION TITLE
what are the .hxc files?

### DISCUSSION THREAD

**Discussion by Shiho-cmd** [FunkinCrew/Funkin#4047] [Q&A] [answered]
i'm trying to make a mod in the new fnf update and wanted to add a script in my song but i noticed fnf use this .hxc files but idk what they are and when i try to open it windows don't recognize can someone help me with this?

**Comment by biomseed** [comment]
https://polymod.io/docs/scripted-classes/
they are hscript files for scripted classes in polymod

**Comment by biomseed** [comment, marked as answer]
hscript is basically a scripting language but with the same syntax as haxe, so you don't need to learn anything new for it if you know haxe

**Comment by Shiho-cmd** [comment]
> hscript is basically a scripting language but with the same syntax as haxe, so you don't need to learn anything new for it if you know haxe

thanks helped me a lot!!!
